roles/automation_services_catalog_client/templates/proxy.conf.j2

```diff
@@ -0,0 +1,4 @@
+[Service]
```

Comment:
Same here, say we don't provid them, file ends up looking like

```
[Service]
Environment="HTTPP_PROXY="
```

Is that valid/accurate ?